sentinel-core/src/main/java/com/alibaba/csp/sentinel/slots/block/flow/FlowRuleManager.java

```diff
@@ -56,13 +56,14 @@
     private static SentinelProperty<List<FlowRule>> currentProperty = new DynamicSentinelProperty<List<FlowRule>>();
 
     @SuppressWarnings("PMD.ThreadPoolCreationRule")
-    private static final ScheduledExecutorService SCHEDULER = Executors.newScheduledThreadPool(1,
+    private static final ScheduledExecutorService SCHEDULER = Executors.newScheduledThreadPool(2,
         new NamedThreadFactory("sentinel-metrics-record-task", true));
 
     static {
         flowRules.set(Collections.<String, List<FlowRule>>emptyMap());
         currentProperty.addListener(LISTENER);
         startMetricTimerListener();
+        SCHEDULER.scheduleAtFixedRate(new MetricBeanTimerListener(), 0, 1, TimeUnit.SECONDS);
     }
```

Comment:
Now there are two thread to execute, is there data inaccuracy in multithreading?
Tried with `FlowQpsDemo`, the `PassQps`,`BlockQps` are always 0 in jconsole.

`com.alibaba.csp.sentinel.node.metric.MetricWriter#write` 
`com.alibaba.csp.sentinel.node.metric.jmx.MetricBeanWriter#write`
When the two operations are put in a same thread, it works fine and got the expected result in jconsole.

Maybe that's the reason, as the comment mentioned in `com.alibaba.csp.sentinel.node.StatisticNode#metrics`,
**The fetch operation is thread-safe under a single-thread scheduler pool.**
```java
@Override
public Map<Long, MetricNode> metrics() {
    // The fetch operation is thread-safe under a single-thread scheduler pool.
    long currentTime = TimeUtil.currentTimeMillis();
    currentTime = currentTime - currentTime % 1000;
    ...

    return metrics;
}
```

Reply:
thank you to review my code, I will check the codes and sovle it.